account_credit_control/README.rst

```diff
@@ -16,14 +21,24 @@ You are able to specify a particular policy for one partner or one invoice.
 Usage
```

Comment:
There isn't an Accounting tab in company form now. There is a "General information" tab,